src/js/node/Fs.hx

```diff
@@ -695,12 +703,12 @@ extern class Fs {
 		`mode` defaults to 0777.
 	**/
 	@:overload(function(path:FsPath, callback:Error->Void):Void {})
-	static function mkdir(path:FsPath, mode:FsMode, callback:Error->Void):Void;
+	static function mkdir(path:FsPath, options:FsMakeDirOptions, callback:Error->Void):Void;
```

Comment:
> The optional options argument can be an integer specifying mode (permission and sticky bits), or an object with a mode property and a recursive property indicating whether parent directories should be created. 

Shouldn't we do `EitherType<FsMode, FsMakeDirOptions>`?

Reply:
Actually, there's more to it:

> The callback is given a possible exception and, if recursive is true, the first directory path created, (err, [path]).

and

> Returns undefined, or if recursive is true, the first directory path created. 

for `mkdirSync`

It's a bit tricky to have proper typing for this, so we can ignore this for now, but we'll have to look into this at some point. Maybe some `@:overload` trick can be used for that.